geometry-validation/vgapp/vgapp.cc

```diff
@@ -24,6 +25,9 @@ struct Volume
 {
     std::string volume_name;
     std::string material_name;
+    int numPlaced = 0;
+    int copyNumMin = 9999;
```

Comment:
```suggestion
    int min_copy_num = std::numeric_limits<int>::digits10;
```

And use same snake_case convention for the the other 2 variables.
```suggestion
    int copyNumMin = 9999;
```
```suggestion
    int copyNumMin = 9999;
```